release/models/network-instance/openconfig-network-instance-l2.yang

```diff
@@ -376,4 +383,308 @@ submodule openconfig-network-instance-l2 {
+                  type leafref {
+                    path "../state/producer";
+                  }
+                  description
+                    "Reference to producer list key";
+                }
+
+                container state {
+                  config false;
+                  description "State container for L2RIB MAC Table Entries";
+                  uses l2ni-l2rib-common-producer-state;
+                  uses l2ni-l2rib-mac-table-producer-state;
+                }
+              }
+            }
+
+            uses l2ni-l2rib-common-next-hop-state;
+          }
+        }
+
+      }
+
+      container mac-ip-table {
+        description
+          "Oper state container for MAC-IP address information installed into
+           the MAC VRF of the L2RIB";
+
+        container entries {
+          description
+            "Enclosing container for list of MAC-IP address entries";
+
+          list entry {
+            key "mac-address vlan host-ip";
+            description "List of learned MAC-IP addresses";
+
+            leaf mac-address {
+              type leafref {
+                path "../state/mac-address";
+              }
+              description "Leafref of MAC-IP address object";
+            }
+
+            leaf vlan {
+              type leafref {
+                path "../state/vlan";
+              }
+              description
+                "Leafref VLAN on which the MAC-IP address is present";
+            }
+
+            leaf host-ip {
+              type leafref {
+                path "../state/host-ip";
+              }
+                description "Leafref host-ip for the MAC-IP table entry";
+            }
+
+            container state {
+              config false;
+              description
+                "Operational state data for L2RIB MAC-IP table object entry";
+              uses l2ni-l2rib-mac-ip-table-state;
+            }
+
+            container producers {
+              description "Source producers for each MAC-IP Table entry";
+              list producer {
+                key "producer";
+                description
+                  "List of producers for each MAC-IP table entry";
+
+                leaf producer {
+                  type leafref {
+                    path "../state/producer";
+                  }
+                  description
+                    "Reference to producer list key";
+                }
+
+                container state {
+                  config false;
+                  description "State container for L2RIB MAC Table Entries";
+                  uses l2ni-l2rib-common-producer-state;
+                }
+              }
+            }
+
+            uses l2ni-l2rib-common-next-hop-state;
+          }
+        }
+      }
+    }
+  }
+
+  grouping l2ni-l2rib-mac-table-state {
+    description "L2RIB MAC Table Oper State Grouping";
+    uses l2ni-l2rib-common-state;
+  }
+
+  grouping l2ni-l2rib-mac-ip-table-state {
```

Comment:
Please use the grouping naming convention:

> Grouping naming 
Grouping names should make it easy to quickly understand the nature of the data within. A suggested convention is xxx-yyy[-config|state|top], where xxx is the top-level module name (without the openconfig prefix), yyy is a string which indicates the contents of the groupings.

https://github.com/openconfig/public/blob/master/doc/openconfig_style_guide.md#grouping-naming

Reply:
@sbarguil I was trying to follow the same convention that is already used by other groupings in this file `openconfig-network-instance-l2.yang`.

The other groupings are:

* grouping l2ni-instance
* grouping l2ni-instance-common-config
* grouping l2ni-fdb-mac-config
* grouping l2ni-encapsulation-config
* grouping l2ni-mac-table-state

Reply:
@sbarguil and I discussed this and we will leave it as is since it follows the convention already defined in the yang file.